src/azure-cli-core/azure/cli/core/adal_authentication.py

```diff
@@ -75,6 +76,13 @@ def get_token(self, *scopes, **kwargs):  # pylint:disable=unused-argument
         logger.debug("AdalAuthentication.get_token invoked by Track 2 SDK with scopes=%s", scopes)
 
         _, token, full_token, _ = self._get_token(_try_scopes_to_resource(scopes))
+
+        try:
+            expires_on = full_token.get('expiresOn', full_token['expires_on'])
```

Comment:
`full_token['expires_on']` will be evaluated first causing failure.

```suggestion
            expires_on = full_token['expiresOn']
```

Reply:
This will also break Cloud Shell because `expires_on` is an epoch time and should be used directly. Parsing it will result in failure.